src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -0,0 +1,60 @@
+import { Flex } from "@artsy/palette"
+import Mapbox from "@mapbox/react-native-mapbox-gl"
+import { LocationMap_location } from "__generated__/LocationMap_location.graphql"
+import React from "react"
+import { NativeModules } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components/native"
+import { FiltersBar } from "./Components/FiltersBar"
+const Emission = NativeModules.Emission || {}
+
+Mapbox.setAccessToken(Emission.mapBoxAPIClientKey)
+
+const Map = styled(Mapbox.MapView)`
+  height: 100%;
+`
+
+export enum PartnerType {
+  gallery = "Gallery",
+  museum = "Museum",
+  fair = "Fair",
+}
+
+interface Props {
+  location: LocationMap_location
+  partnerType: PartnerType
+  partnerName?: string
+}
+
+export class GlobalMap extends React.Component<Props> {
+  render() {
+    const { lat, lng } = { lat: 40.770424, lng: -73.981233 }
+
+    return (
+      <Flex mb={0.5}>
+        <FiltersBar tabs={["All", "Saved", "Fairs", "Galleries", "Museums"]} />
+        <Map
+          key={lng}
+          styleURL={Mapbox.StyleURL.Light}
+          centerCoordinate={[lng, lat]}
+          zoomLevel={14}
+          logoEnabled={false}
+          attributionEnabled={false}
+        />
+      </Flex>
+    )
+  }
+}
+
+export const GlobalMapContainer = createFragmentContainer(
+  GlobalMap,
+  graphql`
+    fragment GlobalMap_viewer on Viewer
+      @argumentDefinitions(near: { type: "Near", defaultValue: { lat: 22.3964, lng: 114.1095 } }) {
```

Comment:
This is Hong Kong, right? Why is that a default / can you add a comment for that?

Reply:
Yes, I set it to Hong Kong by default for now as that's the city where we're launching the feature but we should remove the default value when the location service is ready

Reply:
Can you elaborate on the "location service" you mentioned here? Is that what will map a devices's currents coords to an appropriate LD city?

Also, I'm wondering if you want to fetch shows the way shown below, or if you want to use a named city entry point, [such as this](https://metaphysics-production.artsy.net/?query=%7B%0A%20%20city(slug%3A%20%22london-united-kingdom%22)%20%7B%0A%20%20%20%20slug%0A%20%20%20%20name%0A%20%20%20%20shows(discoverable%3A%20true%2C%20first%3A%205)%20%7B%0A%20%20%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D)? That root query is intended to be a convenient entry point for LD-city related data (and assumes the device's current position has been mapped to a city slug). If that won't be useful we should discuss.

Reply:
So what I meant by location service is really a client side native library that abstracts getting the user device's location and provides the initial coordinates the map should be centered on. Basically the following pseudo logic

```
  // Logic for whether to show a city or cities list
  //
  // 1. Get user location (lat, lng) from native code
  // 2. do a lookup to get nearest city up to a certain threshold
  // 3. if within threshold select nearest city
  // 4. else show cities list
```

Good thing you mention that though @anandaroop because I forgot with had a `city` root field in MP was gonna bring that up. What would be great is if we also support querying for a city using coordinates such as
```
city(near: $near) {
}
```
which would return null if there is no city nearby and that way we just show the list of cities instead

Reply:
👍 So in that case, the logic to select an appropriate LD city happens server-side over in MP, rather than client side yeah? And the pseudo-logic above becomes something like:

```
1. Get user location (lat, lng) from native code
2. Request `city(near: lat, lng)` from MP
3. If the response is null, or something representing "no nearest city", render the picker
4. Else render the returned data
```

(MP specifics are debatable, but the flow is something like the above?)

Reply:
Yes, exactly the pseudo logic you described